app/templates/components/upf-stat.hbs

```diff
@@ -29,10 +29,10 @@
 </span>
 
 {{#if label}}
-  <span class="upf-stat__label">
+  <span class={{concat 'upf-stat__label' (if tooltip '--with-tooltip')}}>
```

Comment:
use double quotes here :)

Reply:
and maybe
`if tooltip "upf-stat__label--with-tooltip" "upf-stat__label"`
is clearer, but debatable 🤷 ^^

Reply:
or 
`class="upf-stat__label{{if tooltip "--with-tooltip"}}"` :trollface:

Reply:
Fixed for your solution @Miexil